<launch>

    <include file="$(find farmaidbot)/launch/camera_rect.launch"/>
    <include file="$(find farmaidbot)/launch/tag_detection.launch"/>

    <node pkg="farmaidbot" type="apriltag_localization.py" name="apriltag_localization" output="screen">
        <param name="param_path" value="$(find farmaidbot)/parameters/static_poses.yaml"/>
    </node>

</launch>
